laser_filters_plugins.xml

```diff
@@ -60,6 +60,12 @@
 	 This is a filter that removes points on directions defined in a mask from a laser scan.
       </description>
     </class>
+    <class name="laser_filters/LaserScanMedianSpatialFilter" type="laser_filters::LaserScanMedianSpatialFilter" 
+	    base_class_type="filters::FilterBase&lt;sensor_msgs::msg::LaserScan&gt;">
+      <description>
+	This is a spatial 1D median filter which filters sensor_msgs::msg::LaserScan messages.  
```

Comment:
Could you expand the description a bit to explain when this is useful? For example that it removes outliers and smooths, as you say in the PR description.

Reply:
Expanded description.